website/static/js/search.js

```diff
@@ -144,11 +144,19 @@
                 message: '<h4>Queries</h4>'+
                     '<p>Search uses the <a href="http://extensions.xwiki.org/xwiki/bin/view/Extension/Search+Application+Query+Syntax">Lucene search syntax</a>. ' +
                     'This gives you many options, but can be very simple as well. ' +
-                    'Examples of valid searches include:' +
-                    '<ul><li><a href="/search/?q=repro*">repro*</a></li>' +
-                    '<li><a href="/search/?q=brian+AND+title%3Amany">brian AND title:many</a></li>' +
-                    '<li><a href="/search/?q=tags%3A%28psychology%29">tags:(psychology)</a></li></ul>' +
-                    '</p>'
+                    'Examples of valid searches include:</p>' +
+                    '<ul><li><code><a href="/search/?q=repro*">repro*</a></code></li>' +
+                    '<li><code><a href="/search/?q=brian+AND+title%3Amany">brian AND title:many</a></code></li>' +
+                    '<li><code><a href="/search/?q=tags%3A%28psychology%29">tags:(psychology)</a></code></li></ul>' +
+                    '<h4>Special Characters</h4>' +
+                    '<p>When manually inputting a query, the following characters must be escaped:' +
+                    '<p>' +
+                    $.map(
+                        '+-&|!(){}[]^"~*?:\\'.split(""),
+                        function(x) { return '<code>' + x + '</code>' }
```

Comment:
I'm not seeing it, but this code has also changed significantly.

Here's a link to the code as it stands right now: https://github.com/lyndsysimon/osf.io/blob/cca2977865bacb5f6b095548aa4b77f468f27e8d/website/static/js/search.js#L159-L179